pina/model/network.py

```diff
@@ -0,0 +1,95 @@
+import torch.nn as nn
+import torch
+from pina.label_tensor import LabelTensor
+
+
+class Network(nn.Module):
+    """The PINA implementation of any neural network.
+
+    :param iterable(torch.nn.Module) model: the torch model of the network
```

Comment:
the input is a torch.nn.Module, not an iterable

Reply:
Also add the final dot at the end of the sentence

Reply:
this small changes have been modified. I added also test for extra feature in case are passed to `Network`